server/service/src/location/insert.rs

```diff
@@ -20,6 +20,7 @@ pub struct InsertLocation {
     pub name: Option<String>,
     pub on_hold: Option<bool>,
     pub location_type_id: Option<String>,
+    pub volume: f64,
```

Comment:
Would make this optional and unwrap to 0.0.. we have plugins and other things using the api.. doubt anything is inserting locations right now but nice to be on the more cautious side